rocky: Update OVN to latest 24.03 version

priteau opened the pull request:
This repository provides ovn24.03-24.03.2-34 which includes some bug fixes [1].

[1] https://git.centos.org/rpms/ovn/blob/c9s-sig-nfv-ovn24.03/f/SPECS/ovn24.03.spec#_534

mnasiadka commented on releasenotes/notes/ovn-24.03.2-34-3d64e85ee8ccdc47.yaml:
```diff
@@ -0,0 +1,5 @@
+fixes:
+  - |
+    OVN packages in Rocky Linux 9 container images have been updated to the
+    latest minor release in the ``24.03`` series: ``ovn24.03-24.03.2-34``.
```
I guess it makes sense to mention Neutron update as well?

priteau committed:
rocky: Update OVN to latest 24.03 version

This repository provides ovn24.03-24.03.2-34 which includes some bug
fixes [1].

[1] https://git.centos.org/rpms/ovn/blob/c9s-sig-nfv-ovn24.03/f/SPECS/ovn24.03.spec#_534
